examples/headless/Cargo.toml

```diff
@@ -20,3 +20,6 @@ pollster = { workspace = true }
 env_logger = "0.11.5"
 png = "0.17.14"
 futures-intrusive = { workspace = true }
+
+[features]
+cosmic_text = ["scenes/cosmic_text"]
```

Comment:
Our style is to have a final newline at the end of all files.
(This also applies to the other files where you've chosen to omit this.)

Reply:
This appears to have re-regressed in the same place.